test/client/deployStrategies/auraDeploy.test.ts

```diff
@@ -10,13 +10,14 @@ import { AuthInfo, Connection } from '@salesforce/core';
 import { MockTestOrgData, testSetup } from '@salesforce/core/lib/testSetup';
 import { expect } from 'chai';
 import * as fs from 'graceful-fs';
-import { RecordResult } from 'jsforce';
+import { SaveError, SaveResult } from 'jsforce';
 import { createSandbox, SinonSandbox } from 'sinon';
+import { AnyJson } from '@salesforce/ts-types';
 import { nls } from '../../../src/i18n';
-import { ToolingDeployStatus, ComponentStatus } from '../../../src/client';
```

Comment:
is there an eslint rule for alphabetizing imports that we can turn on to have less of this happen?

Reply:
there is... but I couldn't get it working to match what my IDE did. I'll turn off that setting